frontend/packages/console-app/src/components/nodes/popovers/MarkAsSchedulablePopover.tsx

```
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { Button } from '@patternfly/react-core';
+
+import { PopoverStatus } from '@console/shared';
+import { NodeKind } from '@console/internal/module/k8s';
+import NodeStatus from '../NodeStatus';
+import { makeNodeSchedulable } from '../../../k8s/requests/nodes';
+
+const MarkAsSchedulablePopover: React.FC<MarkAsSchedulablePopoverProps> = ({ node }) => {
+  const onClickMarkAsSchedulable = async () => {
+    try {
+      await makeNodeSchedulable(node);
+    } catch (err) {
+      console.error(err); // eslint-disable-line no-console
```

We want to surface this error to the user. Maybe use `errorModal` here?